satpy/etc/readers/amsr2_l2.yaml

```diff
@@ -0,0 +1,41 @@
+reader:
+  name: amsr2_l2
+  description: GCOM-W1 AMSR2 instrument HDF5 reader
+  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
+  sensors: [amsr2]
+
+file_types:
+  amsr2_l2:
+    file_reader: !!python/name:satpy.readers.amsr2_l2.AMSR2L2FileHandler
+    file_patterns: ['{platform_shortname:3s}{instrument_shortname:3s}_{start_time:%Y%m%d%H%M}_{path_number:3d}{orbit_direction:1s}_L2{process_kind:2s}{product_id:3s}{resolution_id:1s}{dev_id:1s}{product_version:1s}{algorithm_version:3d}{parameter_version:3d}.h5']
+
+datasets:
+  longitude_ssw:
+    name: longitude_ssw
+    file_type: amsr2_l2
+    standard_name: longitude
+    resolution: 15000
+    units: degree
```

Comment:
Should be `degrees_east` and latitude units should be `degrees_north`.

Reply:
will do